.github/workflows/roadmap_automation.yml

```diff
@@ -0,0 +1,109 @@
+name: public-roadmap
+
+on:
+  issues:
+    types: [demilestoned, milestoned]
+
+jobs:
+  # for debugging purpoeses
+  event_data:
+    runs-on: ubuntu-latest
+    steps:
+      - name: echo
+        env:
+          GITHUB_PAYLOAD: ${{ toJson(github) }}
+        run: |
+          echo "$GITHUB_PAYLOAD"
+
+
+  add_to_public_roadmap:
+    runs-on: ubuntu-latest
+    if: github.event.action == 'milestoned'
+    steps:
+      - name: Fetch project data
```

Comment:
We'll have to hard-code some data (project name or project number). Now, since we'll have to do that anyway, we can simply fetch the project ID and hard-code that one. That way we can get rid of this step completely.

Reply:
good call.